src/components/registerV2/confirmPassphrase.css

```diff
@@ -126,9 +127,9 @@
 
   & .hasErrors {
     & .choice {
-      background-color: var(--color-error);
+      border: solid 1px var(--color-error);
```

Comment:
use the sienna-burnt color for errors, you may want to update on the variables file instead.